woocommerce/woocommerce-android
# Add Inventory to Variation details

Fixes #2520.

![image](https://user-images.githubusercontent.com/1522856/85860811-a1d5c780-b7bf-11ea-86dc-73369b8cb183.png)

**To test**
1. Go to products
2. Tap on a variable product
3. Select a variation with the inventory set
4. Verify it's correctly displayed
